From 5f26fb0bf25cac8cddb2b8de539c841ee6ad6696 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 09 Dec 2022 11:34:41 +0200
Subject: [PATCH] [SW-0] gaudi3: remove c2c pll configuration for pldm

Currently, mesh, nch and c2c pll configuration is done as part of
network pll configuration to speed up dma copy of preboot image and
execution of preboot on sram. It is identified that only mesh
and nch pll configuration is required to copy preboot on sram
by SPI DMA correctly (w/o SPI DMA FIFO) so removing c2c pll
configuration.

Change-Id: I3ef57d6d75dc7422f975a42f5a63d9f6f2e06652
---

diff --git a/arch/gaudi3/arc/pldm.c b/arch/gaudi3/arc/pldm.c
index fd1a769..ae7efbf 100644
--- a/arch/gaudi3/arc/pldm.c
+++ b/arch/gaudi3/arc/pldm.c
@@ -39,26 +39,6 @@
 
 static struct hl_pll_cfg gaudi3_nw_pll_cfg[] = {
 	{
-		.name = "G1C2C", .base = mmD0_G1_C2C_PLL_CTRL_BASE,
-		.nr = 1, .nf = 224, .od = 6, .nb = 0,
-		.div_factor = {0, 1, 1, 7},
-		.div_sel = {HL_PLL_PLL_CLK,
-			HL_PLL_DIV_PLL,
-			HL_PLL_DIV_PLL,
-			HL_PLL_DIV_PLL
-		},
-	},
-	{
-		.name = "G5C2C", .base = mmD0_G5_C2C_PLL_CTRL_BASE,
-		.nr = 1, .nf = 224, .od = 6, .nb = 0,
-		.div_factor = {0, 1, 1, 7},
-		.div_sel = {HL_PLL_PLL_CLK,
-			HL_PLL_DIV_PLL,
-			HL_PLL_DIV_PLL,
-			HL_PLL_DIV_PLL
-		},
-	},
-	{
 		.name = "G4NCH", .base = mmD0_G4_NCH_PLL_CTRL_BASE,
 		.nr = 1, .nf = 224, .od = 6, .nb = 0,
 		.div_factor = {0, 1, 1, 7},
